### DISCUSSION TITLE
Windows ARM64 support?

### DISCUSSION THREAD

**Discussion by holaimscott** [QL-Win/QuickLook#1121] [Q&A] [answered]
Hi, this might be something I might have not looked into well, but I was trying to double check if Quicklook has native Windows ARM support? Would the .MSI or .ZIP files work right our of the box on ARM natively? (of couse after unblocking the ZIP file in that one option)

Thank you!

**Comment by xupefei** [comment, marked as answer]
Currently QL is compiled against x86 and x64, and there're hook modules that might depend on a specific layout of ASM code. It might be worth trying to run QL on ARM64 using the native translation layer, but I don't have a device to test.

**Reply by holaimscott** [reply]
Ahh I see, thank you! I will ask some folks that have ARM devices to test and see if it works properly... I would be neat to see in future releases native ARM support so that it can work without much overwork (and since alot of ARM PCs are on the rise)

**Reply by craigcampbell** [reply]
I have a arm device to test if you want to send me a build

**Comment by MarathonMax** [comment]
Looks like an old thread here, but wanted to ask is QL compatible with new ARM devices like SurfacePro SnapDragon? Don't think so.

**Reply by carucio** [reply]
It does (Dell XPS13 9345) and it's fast as always even if running translated from x64. I only wish it would use less RAM

**Reply by MarathonMax** [reply]
my mistake ... it DOES run on my Surface Pro Snapdragon
